spec/openapi.yaml

```diff
@@ -3950,6 +3997,299 @@ components:
+                      type: string
+                      format: uuid
+                    firstName:
+                      description: First name
+                      type: string
+                    lastName:
+                      description: Last name
+                      type: string
+                    company:
+                      description: Company name
+                      type: string
+                      nullable: true
+                    streetName:
+                      description: Street name
+                      type: string
+                    streetNumber:
+                      description: Street number
+                      type: string
+                    streetSuffix:
+                      description: Street number suffix
+                      type: string
+                      nullable: true
+                    zipCode:
+                      description: Zip code
+                      type: string
+                    city:
+                      description: City
+                      type: string
+                    countryCode:
+                      description: ISO 3166-1 alpha-2 country code
+                      type: string
+                      pattern: '^[A-Z]{2}$'
+                    phone:
+                      description: Phone number
+                      type: string
+                      nullable: true
+                    gender:
+                      description: Gender
+                      type: string
+                      nullable: true
+                      enum:
+                        - F
+                        - M
+        properties:
+          description: Transaction properties
+          type: array
+          items:
+            type: string
+    PaymentMutation:
+      example:
+        id: "1"
+        type: "authorize"
+        reference: null
+        price: "499"
+        description: null
+        createdAt: "2021-06-29T14:41:26+02:00"
+        expiresAt: null
+      properties:
+        id:
+          description: Mutation ID
+          type: number
+        type:
+          description: Mutation type
+          type: string
+          enum:
+            - "authorize"
+            - "capture"
+            - "complete_authorize"
+            - "complete_purchase"
+            - "fail"
+            - "notify"
+            - "purchase"
+            - "refund"
+            - "void"
+        reference:
+          description: Optional mutation reference
+          type: string
+          nullable: true
+        price:
+          description: Price (in cents)
+          type: number
+        description:
+          description: Optional mutation description
+          type: string
+          nullable: true
+        createdAt:
+          description: Mutation creation date
+          type: string
+          format: date-time
+        expiresAt:
+          description: Mutation expiration date
```

Comment:
This one can be explicitly named 'Optional' as well, just like `description` and `reference`, for consistency. Or alternatively the term 'optional' can be removed from all since the nullable property already implicitly says those are optional.